subprojects/docs/src/docs/release/notes.md

```diff
@@ -170,6 +170,14 @@ The following are the features that have been promoted in this Gradle release.
 - `Project.normalization`
 - `GroovyCompile.groovyCompilerJvmVersion` and `javaToolChain` properties
 - `JavaVersion.VERSION_11` constant along with `isJava11()` and `isJava11Compatible()` methods
+- all pre 4.0 incubating APIs have been promoted
+- parallel task execution
+- continuous build
+- composite build
+- the `java-gradle-plugin` plugin
+- the `distribution` plugin
+- the `jacoco` plugin
+- the `build-init` plugin
```

Comment:
These items strike me as more important than the ones above. Shall we move them to the beginning of the list?

Reply:
Moved. I also adjust `standard_plugins.adoc` to reflect the de-incubations.